lib/iris/fileformats/pp_rules.py

```diff
@@ -597,9 +597,12 @@ def date2hours(t):
         coords_and_dims.append((DimCoord(t1_epoch_hours, standard_name='time', units=epoch_hours_unit), None))
         coords_and_dims.append((DimCoord(t2_epoch_hours, standard_name='forecast_reference_time', units=epoch_hours_unit), None))
 
+    # Don't add time coordinates when years are zero, months match, and
+    # lbtim.ib == 2. These are handled elsewhere.
     if \
             (lbtim.ib == 2) and \
             (lbtim.ic in [1, 2, 4]) and \
+            (not (t1.year == 0 and t2.year == 0 and t1.month == t2.month)) and \
```

Comment:
if we decide on this behaviour, a third test case would be useful, for the 'ignore the times' case described above